scripts/setup/upgrade-postgresql

```diff
@@ -21,6 +21,11 @@ if [ "$UPGRADE_TO" = "$UPGRADE_FROM" ]; then
     exit 1
 fi
 
+if [[ "$UPGRADE_TO" -lt "$UPGRADE_FROM" ]]; then
```

Comment:
Is there a potential issue with this doing string comparisons rather than numeric comparisons? I guess now that we're done with the `9.x` series, the answer is probably no -- since digit boundaries like "10" < "9" is the main class of problem there.

Reply:
`-lt` is a numeric comparison. (It might have failed with `[ 9.9 -lt 9.10 ]` though.)